### PR TITLE
SwiftUI View: Stepper

### PR DESCRIPTION
**Pull request by cilippofilia**
### Description

Stepper is a great tool when giving user granulated control over a specific range of numbers

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by cilippofilia** [comment]
@SSwiniarski @yangc95 @KTom101 can someone start reviewing this PR please?

File: content/swiftui/concepts/views/terms/stepper/stepper.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+var body: some View {
+    Stepper(value: $numberOfPeople, in: 2...10, step: 1) {
+        Text("People count: \(numberOfPeople)")
+    }
+    .padding()
+}
+
+```
+
+- The first parameter that `Stepper` takes is `value:`. This parameter needs to be bound to a variable that tracks changes.
```
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/stepper/stepper.md:44]: This bulleted list should be moved to the end of the syntax section.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cilippofilia, I did a first review and left a comment.

**Review by SSwiniarski — APPROVED** [review]
@cilippofilia looks good for a second review.

**Review by yangc95 — APPROVED** [review]

**Comment by lucwhite** [comment]
Hello @cilippofilia,

I hope this message finds you well! My name is Lucas, and I work on the Curriculum team at Codecademy. Our team is considering ways to encourage more individuals to contribute to Codecademy Docs and improve the contribution experience. I was wondering if you would be willing to participate in an interview to provide information on your background and experience with Docs contribution?

The interview would take about an hour and be conducted over Zoom. Should you agree to participate, we will find a time window using Calendly that works for you within the next three weeks. Let us know if you are interested in participating in an interview or have any questions by responding with your email address or emailing me at the address below!

Best,
Lucas White
lucas@codecademy.com

**Comment by cilippofilia** [comment]
Hey Lucas,

Absolutely! I am available tuesdays, wednesdays, thursays all day. Any
other day before 11am.

Have a great weekend!

Filippo

**Comment by cilippofilia** [comment]
Hey Lucas,

Absolutely! I am available tuesdays, wednesdays, thursays all day. Any
other day before 11am.

Have a great weekend!

Filippo

**Pull request merged**